api/catalog/settings.py

```diff
@@ -376,3 +376,5 @@
 MAX_ANONYMOUS_PAGE_SIZE = 20
 MAX_AUTHED_PAGE_SIZE = 500
 MAX_PAGINATION_DEPTH = 20
+
+BASE_URL = "https://wordpress.org/openverse/"
```

Comment:
It would be nice to be able to configure this variable from the environment:
```suggestion
BASE_URL = config("BASE_URL", default="https://wordpress.org/openverse/")

```

Reply:
Thank you @krysal for the suggestions!